index.js

```diff
@@ -50,40 +51,90 @@ app.get('/restaurant', (req, res) => {
       })
     }
   }
-  res.json(response)
+  res.json(response);
+
 })
 
 app.post('/restaurant', (req, res) => {
-    if (!req.body.name || !req.body.latitude || !req.body.longitude) {
-        res.json({
-            "code": 404,
-            "status": "Failed",
-            "message": "Missing name, latitude or longitude"
-        })
+  if (!req.body.name || !req.body.latitude || !req.body.longitude) {
+    res.json({
+      "code": 404,
+      "status": "Failed",
+      "message": "Missing name, latitude or longitude"
+    })
+  }
+  else {
+    let restoID = backend.createRestaurant(req.body.name, req.body.latitude, req.body.longitude)
+    let restaurants = backend.getRestaurants()
+    res.json(restaurants[restoID])
+  }
+})
+
+// ZOMATO DAILY MENU DATA MODEL
+// {
+//   "daily_menu": [
+//     {
+//       "dishes": [
+//         {
+//           "dish_id": "104089345",
+//           "name": "Tatarák ze sumce s toustem",
+//           "price": "149 Kč"
+//         }
+//       ]
+//     }
+//   ]
+// } 
+app.get('/dailymenu', (req, res) => {
+
+  let restaurants = backend.getRestaurants();
+  let response = []
+
+  if (req.query.res_id) {
+    if (!restaurants[req.query.res_id]) {
+      response = {
+        "code": 404,
+        "status": "Not Found",
+        "message": "Not Found"
+      }
     }
     else {
-        let restoID = backend.createRestaurant(req.body.name, req.body.latitude, req.body.longitude)
-        let restaurants = backend.getRestaurants()
-        res.json(restaurants[restoID])
+      restaurants[req.query.res_id].dishes.forEach(item => {
+        response.push({
+          dishes: {
+            dish_id: item.dish_id,
+            name: item.burgerName,
+            price: item.price
+          }
+        })
+      })
     }
+  }
+
```

Comment:
We're missing an `else` clause here to match the `if` on line 92. If the user doesn't provide a `res_id` at all, it should return an error

Reply:
ok